install/collecting_ncn_mac_addresses.md

```diff
@@ -34,7 +34,7 @@ boot-check nodes to dump network device information without an operating system.
 
 1. (`pit#`) Modify the boot so that nodes stop network booting after dumping their network devices.
 
-    > **NOTE:** Removing the iPXE script will prevent network booting. Be aware that the
+    > ***NOTE*** Removing the iPXE script will prevent network booting. Be aware that the
```

Comment:
What do you have against colons? I think normally a colon would belong there...

Reply:
From the standpoint of inventing a "tag" the colon wasn't really symmetrical as far as formatting went.